src/libraries/System.Runtime/tests/System.Runtime.InteropServices.RuntimeInformation.Tests/DescriptionNameTests.cs

```diff
@@ -196,17 +196,8 @@ public void VerifyFrameworkDescriptionContainsCorrectVersion()
                 return;
 
             Assert.DoesNotContain("+", version); // no git hash
-
-#if STABILIZE_PACKAGE_VERSION
```

Comment:
I think this should stay as is. It is a regression test for a bug that slipped through the cracks some time ago.